aldryn_forms/models.py

```diff
@@ -17,6 +19,7 @@
 from .helpers import is_form_element
 
 
+USES_TREEBEARD = LooseVersion(cms.__version__) >= LooseVersion('3.1')
```

Comment:
I don't agree, we could just check if treebeard is in installed apps but that's not accurate because both could be running. we can safely assume that if cms is 3.1 or up then we're using treebeard, maybe the variable name is misleading. We can rename variable to CMS_3_1_UP